#goal definiton
string prefix 
---
#result definition
float32 distance
int32 numFeatures
int32 numMaps
---
#feedback
string fileName
float32 distance 
int32 numFeatures
int32 numberOfMaps
int32 mapIndex
bool mapLoaded
